File: Primeiro período/AED-1/Java/Trabalho Prático 01 - AED1/Grau.java
// Autor: Luiz Junio <luisjuniorbr@gmail.com>

import java.util.Scanner;

class Grau { 
   public static void main (String [] args)
   {
      Scanner s = new Scanner (System.in);
      System.out.print("Digite escreva um grau em farenheit: ");
      double grauf = s.nextDouble();
      double grauc = 5.0/9 * (grauf - 32);
      System.out.printf("O grau em celcius é: %.2f°C ",grauc);
      
      
       
   }
}      

File: Primeiro periodo/AED-1/CSharp/Trabalho Prático 01/Grau.cs
using System;

public class Grau {
    public static void Main(){
        Console.Write("Digite um grau em fahrenheit: ");
        double grauf = Convert.ToDouble(Console.ReadLine());
        double grauc = 5.0 / 9 * (grauf - 32);
        Console.WriteLine("O grau em celcius é: {0:F2}°C", grauc);
    }
}